developers/contributing_how_tos.md

```diff
@@ -0,0 +1,41 @@
+# How to contribute a napari How-To
+
+This should be a sentence (or brief paragraph) about what this how-to will achieve and how it helps address a common problem/task/need.
```

Comment:
what the how-to will achieve doesn't feel super clear to me, is it about what the document itself will demonstrate? Maybe demonstrate is a clearer word (or I'm just slow 🙂)

Reply:
This can be stated as what the reader will be able to do after reading through the document. I like to focus these statements in terms of the reader and not the document. So, I wouldn't say "what the how-to will achieve" but "what the reader will achieve after reading this how-to" (maybe too subtle but I feel it helps to pinpoint the goal of each document).